### ISSUE TITLE
Column types with wrong HTML fields or lack of validation

### ISSUE THREAD

**Issue by germain-italic** [jan-vandenberg/cruddiy#75]
These types are handled incorrectly:
- a_timestamp TIMESTAMP
- a_char CHAR(10)
- a_binary BINARY(10)
- a_json JSON
- a_point POINT
- a_linestring LINESTRING
- a_polygon POLYGON

[Here](https://github.com/jan-vandenberg/cruddiy/pull/74/commits/3b53e3e074b5f96d40bf3341c9997efce54a43c1) is a table schema for all field types.

**Comment by jan-vandenberg** [comment]
I think I can close this one?

**Comment by germain-italic** [comment]
No, #74 is only a fix for undefined indexes, but does not handle the types abovementioned

**Comment by germain-italic** [comment]
Just found out this related issue in PR #107
